tests/utils/deployments.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+)
+
+func WaitForDeploymentReplicasReadyOrDie(c *kubernetes.Clientset, namespace, name string) {
+	if err := WaitForDeploymentReplicasReady(c, namespace, name); err != nil {
```

Comment:
We make sure before we run the tests in STDCI that the http server is up and running. So this won't do much, except if we are running on a separate cluster or something.

Reply:
@awels Okay, should we have consideration for non-STDCI environments?  e.g. dev's testing changes locally

Reply:
I think it's a mistake to focus cdi testing to only work with Red Hat QE and kubevirt. CDI is an upstream, community project (admittedly mostly/only Red Hat people at the moment), and we need to keep the community aspect in mind. As long as cdi remains separate from kubevirt, we need to look for more generalized solutions, imo.